sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeDisableAutomaticCheckForUpdateTest.java

```diff
@@ -61,4 +62,65 @@ public void manualCheckWhileAutomaticCheckDisabledChecksForUpdate() {
         /* HTTP call done. */
         verify(mHttpClient).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
     }
+
+    @Test
+    public void manualCheckAfterActivityResumedWhileAutomaticCheckDisabledChecksForUpdate() {
+
+        /* Start then call disable automatic check for update after Distribute has started. */
+        Distribute.disableAutomaticCheckForUpdate();
+        start();
+        verify(mHttpClient, never()).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
+        Distribute.getInstance().onActivityResumed(mActivity);
```

Comment:
Swap lines 72 and 73, it would never start a HTTP call in background even when automatic updates are enabled.

Reply:
Good catch, it was a mistake.